index.mjs.js

```diff
@@ -20,7 +14,7 @@ Future.map = map;
 Future.bimap = bimap;
 Future.chain = chain;
 
-var Par = concurrify(Future, never, race, parallelAp);
+var Par = concurrify(Future, never, race, function parallelAp(a, b){ return b._parallelAp(a) });
```

Comment:
I changed `parallelAp` from a static constructor to a standard Action. This saved me writing some extra code.